starter/slack-spring-boot/src/main/kotlin/com/kreait/slack/broker/store/user/UserStore.kt

```diff
@@ -0,0 +1,89 @@
+package com.kreait.slack.broker.store.user
+
+import java.time.Instant
+
+/**
+ * Used to handle App-installations
+ */
+interface UserStore {
+
+    /**
+     * returns a User by its Id
+     */
+    fun findById(id: String): User
+
+    /**
+     * Adds a User to the Database
+     */
+    fun put(vararg users: User)
+
+    /**
+     * returns all users of a Workspace/Team
+     */
+    fun findByTeam(teamId: String): List<User>
+
+    /**
+     * removes a User by its Id
+     */
+    fun removeById(id: String)
```

Comment:
Since slack can never delete users do we need this method?

Reply:
since I use it to update the user, probably yes.

Reply:
well, i should drather create a .update() method that does that internally